### ISSUE TITLE
Tracebacks with "<frozen pkgname.modname>" instead file paths

### ISSUE THREAD

**Issue by joonis** [dashingsoft/pyarmor#1025]
Since Pyarmor 8 we get the following exceptions:

```
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mypackage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "</home/.../mypackage/__init__.py>", line 3, in <module>
  File "<mypackage.__init__>", line 2, in <module>
TypeError: <lambda>() takes 0 positional arguments but 1 was given
```

In Pyarmor 7 it was more beautyful with `<frozen pkgname.modname>`:

```
Python 3.8.10 (default, Mar 13 2023, 10:26:41) 
[GCC 9.4.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import mypackage
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "<frozen mypackage>", line 3, in <module>
  File "<frozen mypackage>", line 2, in <module>
TypeError: <lambda>() takes 0 positional arguments but 1 was given
```

**Comment by jondy** [comment]
Got it, I'll fix it, thanks.

**Comment by jondy** [comment]
Got it, I'll fix it, thanks.

**Comment by jondy** [comment]
Fixed in 8.0.20
